layouts/quote.vue

```diff
@@ -0,0 +1,19 @@
+<script setup>
+	import BaseLayout from './base-layout.vue';
+</script>
+
+<template>
+	<base-layout layout="statement">
+		<blockquote class="quote"><slot /></blockquote>
+	</base-layout>
+</template>
+
+<style scoped>
+	.quote {
+		@apply w-1/2 fixed top-[50%] bg-transparent border-0;
```

Comment:
```suggestion
		@apply w-1/2 fixed top-1/2 bg-transparent border-none;
```

The arbitrary top value is available. For the border I'd rather set the `border-style` to `none` rather than give it a width of `0`.